apps/channels/models.py

```diff
@@ -95,7 +104,7 @@ def channel_identifier_key(self) -> str:
             case self.SLACK:
                 return "slack_channel_id"
             case self.COMMCARE_CONNECT:
-                return "connect_id"
+                return "bot_name"
```

Comment:
The reasoning behind using the bot name is that this will be shown to the user, and we don't want other bots to use the same name, which can become misleading

Reply:
Interesting. I'm not sure about this, or whether the connect channel even needs this uniqueness check. But I do think that you're right in saying that it might be confusing or misleading for users if there are two bots with the same name. I guess let's keep this for now. Please add a comment to the code with some reasoning.

Reply:
I share the sentiment. Comment added [here](https://github.com/dimagi/open-chat-studio/pull/1041/commits/fffac63d6ae45f33a5a378daf61ca54681351a1a)

Reply:
Just wondering if we shouldn't make this name more specific since it will be a global check. Maybe something like `cc_bot_name` or `connect_bot_name` or `commcare_connect_bot_name`

Reply:
https://github.com/dimagi/open-chat-studio/pull/1041/commits/b8d3c9509f249c4574e2880652c3c07cb1b7d02e